### DISCUSSION TITLE
Machine/Tarkov wishlist

### DISCUSSION THREAD

**Discussion by AndrewL97** [Mojave-Sun/mojave-sun-13#2157] [Ideas]
Wishlist for machine project and misc tarkov/gridventory stuff while I am looking at it

### Accepted

SYS - Random loot spawners should have a deferred spawn if inside of a container at world start (crates, medkits, etc.)
[QOL - Separate windows for open containers, so we can move things between easily](https://github.com/Mojave-Sun/mojave-sun-13/discussions/2157#discussioncomment-3523158)
[SYS - Wire panel, to move from the TGUI into a physical object with interactive wires](https://github.com/Mojave-Sun/mojave-sun-13/discussions/2157#discussioncomment-3523176)

### Completed

QOL - Move item rotate from a CTRL click on items to a right click on the grid
QOL - If an object is too wide/tall to fit in a container, attempt to rotate when inserting
SYS - Socket object types that allow objects to be inserted into grids - cannot remove or interact with them otherwise (like a power point)

**Comment by ProfessorPopoff** [comment]
I wish upon a pretty pretty star.... 

There is a separate looting interface for anything not on yourself.

Example: You have your backpack in hand and you're rummaging through it. You loot a container and it pops up a separate tarkov interface, and you can take things from it and put it straight into *your* container

**Reply by AndrewL97** [reply]
Agreed

**Reply by AndrewL97** [reply]
also notice that for this idea, my replies all get sorted nicely underneath instead of in the vomit pile?

**Reply by ProfessorPopoff** [reply]
That's awesome

**Comment by AndrewL97** [comment]
SYS - Wire panel, to move from the TGUI into a physical object with interactive wires

Wires can have their color replaced by cutting them out and inserting new wires

**Reply by AndrewL97** [reply]
"Nodes" show allow labels to be added/removed via a pen to name them

Maybe these start out as codes such as "0xCBT" as a randomized ID - with electronics manuals able to be found
Order of wires is randomized for each machine
If all wires are cut, plays should allow people to rearrange the nodes too

**Comment by AndrewL97** [comment]
Image reference dump for Gridventory buttons/slots

**Reply by AndrewL97** [reply]
Transmitter + misc
![image](https://user-images.githubusercontent.com/22902112/189273861-9d22612a-d73d-4845-b2fc-49e60638fee7.png)

mainframe debouncer, vacuum tube module
![image](https://user-images.githubusercontent.com/22902112/189274050-666be853-29b1-46db-803d-4d66534ffbe3.png)
